### DISCUSSION TITLE
value cannot show in react-select when using setValue

### DISCUSSION THREAD

**Discussion by LouisKwong** [react-hook-form/react-hook-form#6170] [Q&A] [answered]
Hi,

I am new to react-hook-form. I have tried using an existing Codesandbox from a user "mkrtchian".
[https://codesandbox.io/s/react-select-with-react-hook-form-forked-fdlti?file=/src/App.js](https://codesandbox.io/s/react-select-with-react-hook-form-forked-fdlti?file=/src/App.js)

The version of react-hook-form is 6.15.5. "India" is shown after 2000ms.

![image](https://user-images.githubusercontent.com/71380005/128120546-cc73f467-1862-4d23-9eb2-9ad8720f0daf.png)

However, if I upgrade the version to 7.10, "India" does not show after 2000ms. By adding `console.log("select value", value)`, value shows undefined in 7.10.

![image](https://user-images.githubusercontent.com/71380005/128120464-5fbd2865-6c8f-4338-9627-0b2d5d39fe48.png)

Is any changes required when using v7?

Thanks.

**Comment by jorisre** [comment, marked as answer]
Hi you can find a migration guide v6 ➡️ v7 here: https://react-hook-form.com/migrate-v6-to-v7/

Migrated codesandbox
👉🏻    https://codesandbox.io/s/react-select-with-react-hook-form-forked-txtsq?file=/src/App.js

**Reply by LouisKwong** [reply]
I missed the migration guide... Thanks a lot. @jorisre
